chainerx_cc/chainerx/routines/linalg.h

```diff
@@ -13,4 +15,14 @@ Array Solve(const Array& a, const Array& b);
 
 Array Inverse(const Array& a);
 
+enum class QrMode {
+    // if K = min(M, N), where `a` of shape (M, N)
+    reduced,  // returns q, r with dimensions (M, K), (K, N) (default)
+    complete,  // returns q, r with dimensions (M, M), (M, N)
+    r,  // returns empty q and r with dimensions (0, 0), (K, N)
+    raw  // returns h, tau with dimensions (N, M), (K, 1)
```

Comment:
Could you follow the existing enum convention which is to have values named `kReduced`, `kComplete`, etc. ("k" for "constant")?